### PR TITLE
ParensSpacing: initial commit

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: Behance/ruleset.xml [hunk 1]
```diff
@@ -29,9 +29,7 @@
     <exclude name="Generic.Files.LineLength"/>
 
     <!-- Squiz standards that are inherited via PSR2 -->
-    <exclude name="Squiz.ControlStructures.ControlSignature"/>
```
Comment (mikesherov) [inline Behance/ruleset.xml]: this will enforce correct elseif/else/catch location!

File: Behance/ruleset.xml [hunk 2]
```diff
@@ -29,9 +29,7 @@
...
     <exclude name="Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine"/>
-    <exclude name="Squiz.Functions.MultiLineFunctionDeclaration.SpaceAfterFunction"/>
```
Comment (mikesherov) [inline Behance/ruleset.xml]: unnecessarily excluded!

File: tests/Behance/Sniffs/WhiteSpace/ParensSpacingSniffTest.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+class Behance_Sniffs_Whitespace_ParensSpacingSniffTest extends AbstractSniffUnitTest {
+
+  public function getErrorList($testFile) {
+
+    return [
+      3 => 1,
```
Comment (mlebrun) [inline tests/Behance/Sniffs/WhiteSpace/ParensSpacingSniffTest.php]: I guess I just don't full understand what this file is doing?

Comment (mikesherov) [reply tests/Behance/Sniffs/WhiteSpace/ParensSpacingSniffTest.php]: it lists how many errors you'll find in the inc file, by line. 1 error on line 3, 1 error on line 4, 2 errors on line 5 :-)

Comment (mlebrun) [reply tests/Behance/Sniffs/WhiteSpace/ParensSpacingSniffTest.php]: ah, 👍  thanks

**Pull request merged**